public/data/SyedSohan.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Syed Sohan",
+  "bio": "Project Executive - IT | Tech Enthusiast",
+  "avatar": "https://github.com/syedsohan.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/syedsohan",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/syedsohan27",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/syedsohan",
+      "icon": "linkedin"
+    }
+  ]
+}
```

```suggestion
}

```
Could a newline be added at EndOfFile?

---

I can if you think it's really that much essential!

---

It isn't, but it's nice to keep files consistent 😅

---

Added.

---

@Vyvy-vi Added.